packages/tools/pluggable-widgets-tools/scripts/copyright-utils.js

```diff
@@ -0,0 +1,12 @@
+function extractParts(copyright) {
+    const startYearIndex = copyright.indexOf("2005-");
```

Comment:
This is very specific for our widgets. If we want to ship this with piw tools we should think of a more generic way of dealing with this.

Reply:
We maintain our piw and it's only our piw's that have gh actions validating them, right? Meaning if we control the format of the copyright then specificity isn't a problem imo, unless I'm missing something.

Reply:
Regarding end-users who create PIWs using the genrator, the generator won't create the script in `package.json` to validate their copyright, so no problem, right? 😊

Reply:
Lets narrow this conversation down to released piw-tools package?